CHANGELOG.md

```diff
@@ -1,3 +1,142 @@
+# 60.1.0
+
```

Comment:
There's no description of this release?

Reply:
Do you mean naming it "v60.1.0"?  I don't think we normally name our releases, do we?

Reply:
https://github.com/material-components/material-components-ios/blob/develop/contributing/releasing.md#new-features--top-level-description

The release engineer is expected to summarize the release’s contents at the top of the release notes, notably calling out any new features that may have been added.